tests/system/test_system.py

```diff
@@ -711,7 +711,9 @@ def test_backup_workflow(self):
         self.to_drop.append(database)
         operation = database.restore(source=backup)
         restored_db = operation.result()
-        self.assertEqual(self.database_version_time, restored_db.restore_info.backup_info.create_time)
```

Comment:
Is this change necessary in this PR?

Reply:
I don't mind if it is, just wondering if you were fixing a style error.

Reply:
It's not necessary, it was just part of the automatic linting, I've removed it locally